### ISSUE TITLE
strange loss value 'nan', during training

### ISSUE THREAD

**Issue by maliho0803** [shamangary/FSA-Net#4]
30th epoch, after that, the loss always is nan, when I training 'FSA_net_Var_Capsule' model
1761/6120 [=======>......................] - ETA: 4:26 - loss: 3.2613
1762/6120 [=======>......................] - ETA: 4:26 - loss: 3.2612
1764/6120 [=======>......................] - ETA: 4:26 - loss: 3.2609
1765/6120 [=======>......................] - ETA: 4:26 - loss: 3.2609
1766/6120 [=======>......................] - ETA: 4:26 - loss: 3.2609
1768/6120 [=======>......................] - ETA: 4:26 - loss: 3.2609
1770/6120 [=======>......................] - ETA: 4:26 - loss: nan
1771/6120 [=======>......................] - ETA: 4:26 - loss: nan
1772/6120 [=======>......................] - ETA: 4:26 - loss: nan
1773/6120 [=======>......................] - ETA: 4:25 - loss: nan
1774/6120 [=======>......................] - ETA: 4:25 - loss: nan
1775/6120 [=======>......................] - ETA: 4:25 - loss: nan
1776/6120 [=======>......................] - ETA: 4:25 - loss: nan
1777/6120 [=======>......................] - ETA: 4:25 - loss: nan
1779/6120 [=======>......................] - ETA: 4:25 - loss: nan
1780/6120 [=======>......................] - ETA: 4:25 - loss: nan
1781/6120 [=======>......................] - ETA: 4:25 - loss: nan
1782/6120 [=======>......................] - ETA: 4:25 - loss: nan
1784/6120 [=======>......................] - ETA: 4:25 - loss: nan
1785/6120 [=======>......................] - ETA: 4:25 - loss: nan
1786/6120 [=======>......................] - ETA: 4:25 - loss: nan

**Comment by shamangary** [comment]
Sorry. I am a bit busy these days. Could you tell me is this happening on every different model or only this one. Because I don't remember encounter this error on the model before.

**Comment by maliho0803** [comment]
I trained tow models, one is ’SSRNET_mt’ which is ok, another is FSA_net_Var_Capsule
'

**Comment by shamangary** [comment]
I reduce the learning rate from 30 epoch, maybe there is something wrong about the learning rate? What's your tensorflow and keras version?

**Comment by maliho0803** [comment]
TensorFlow ‘1.12.0’; Keras '2.2.4'

**Comment by shamangary** [comment]
Can you try to use Tensorflow 1.10 and train again?

**Comment by maliho0803** [comment]
Thank you very much, I will try

**Comment by CrossEntropy** [comment]
Hi @maliho0803 !
Have you solved this problem? Because I had a similar problem. Thanks!
